certified-connectors/Mobsim Send SMS/README.md

```diff
@@ -0,0 +1,10 @@
+## Conector Mobsim Send SMS
```

Comment:
@jpjsantos please format the readme.md file according to [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). Your readme should contain all the necessary headings.